meta-ov/recipes-apps/ovmenu-ng-skripts/files/download-all.sh

```diff
@@ -7,13 +7,12 @@ if [ ! -d "$USB_PATH" ]; then
 	mkdir -p "$USB_PATH"
 fi
 
-if [ -z "$(ls $DOWNLOAD_PATH/* 2>/dev/null)" ]; then
+if [ -z "$(find "$DOWNLOAD_PATH" -type f 2>/dev/null)" ]; then
         echo "No files found !!!"
 else
-        cp -rv "$DOWNLOAD_PATH/*" "$USB_PATH"
+        find "$DOWNLOAD_PATH" -type f -exec basename {} \; -exec cp "{}" "$USB_PATH" \;
```

Comment:
Keeping the `-v` would be easier and faster than invoking `basename`

Reply:
yes but at least on my 5.7" in portrait mode, that lists the full path and file name is just scrolled off screen.